tests/unit/states/test_environ.py

```diff
@@ -6,14 +6,16 @@
 
 # Import Salt Testing libs
 from tests.support.mixins import LoaderModuleMockMixin
-from tests.support.unit import TestCase
+from tests.support.unit import TestCase, skipIf
 from tests.support.mock import (
     MagicMock,
     patch
 )
 # Import salt libs
 import salt.states.environ as envstate
 import salt.modules.environ as envmodule
+import salt.modules.reg as regmodule
+import salt.utils as utils
```

Comment:
I know the `environ` state above was doing it this way, but we are trying to make these imports more uniform. Can you change this to use the full path? That makes it easier to lint/check for these references in newer branches that have diverged from using the funcs in the `salt.utils.__init__.py` file.